tests/onnx/quantization/test_fast_bias_correction.py

```diff
@@ -0,0 +1,59 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+from typing import List
+
+import numpy as np
+import onnx
+import torch
+
+from nncf.quantization.algorithms.fast_bias_correction.onnx_backend import ONNXFastBiasCorrectionAlgoBackend
+from tests.onnx.quantization.common import get_random_dataset_for_test
+from tests.post_training.test_fast_bias_correction import TemplateTestFBCAlgorithm
+from tests.torch.ptq.helpers import ConvTestModel
+
+
+def get_data_from_node(model: onnx.ModelProto, node_name: str):
+    data = [t for t in model.graph.initializer if t.name == node_name]
+    if data:
+        return onnx.numpy_helper.to_array(data[0])
+    return None
+
+
+class TestTorchFBCAlgorithm(TemplateTestFBCAlgorithm):
```

Comment:
Nevertheless, it is `TestTorchFBCAlgorithm` or `TestONNXFBCAlgorithm`?

Reply:
TestONNXFBCAlgorithm